vapi/library/finder/finder.go

```diff
@@ -206,6 +206,12 @@ func (f *Finder) findLibraries(
 			}
 			result = append(result, findResult{result: *lib})
 		}
+		if len(result) == 0 {
+			lib, err := f.M.GetLibraryByID(ctx, token)
```

Comment:
Hi @dougm,

This can be collapsed to reduce scope. Since the scope of `lib` and `err` are already bound by `if len(result) == 0`, then I understand if you wish to keep things as is to prevent a longer line on 210. Up to you.

Reply:
Hi @akutz, you mean `if lib, err := f.M.GetLibraryByID(ctx, token); err == nil {` ?

I personally tend to avoid that as it can be harder to read at times.  Just always having `if err != nil {` following ends up being more consistent.  The for loop body above is that way too.